app/Exceptions/InvalidCep.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Exceptions;
+
+use Exception;
+
+class InvalidCep extends Exception
```

Comment:
Algum motivo especifico pra não usar o prefixo de "Exception" no nome da classe? Na minha visão, fica bem mais claro o que a classe faz utilizando o mesmo.
```suggestion
class InvalidCepException extends Exception
```